Lib/difflib.py

```diff
@@ -710,11 +710,18 @@ def get_close_matches(word, possibilities, n=3, cutoff=0.6):
     Optional arg cutoff (default 0.6) is a float in [0, 1].  Possibilities
     that don't score at least that similar to word are ignored.
 
+    Optional arg key specifies a function of one argument that is used to 
```

Comment:
Under the impression the trailing whitespace at this and the following line cause travis to complain.

Reply:
I see. Thanks for pointing it out. I'll update this along with the other changes.